src/ParametricOptInterface.jl

```diff
@@ -846,14 +946,29 @@ function is_additive(model::ParametricOptimizer, cp::MOI.ConstraintIndex)
     return true
 end
 
-function parameter_dual_in_affine_constraint(model::POI.ParametricOptimizer, cp::MOI.ConstraintIndex)
-    param_dual_affine_constraint = 0
-    for (ci, param_array) in model.affine_constraint_cache
-        for param in param_array
-            if cp.value == param.variable_index.value
-                cons_dual = MOI.get(model.optimizer, MOI.ConstraintDual(), ci)
-                param_dual_affine_constraint += cons_dual*param.coefficient
-            end
+function parameter_dual_in_affine_constraint(model::POI.ParametricOptimizer, cp::MOI.ConstraintIndex)::Float64
```

Comment:
instead of passing cp you could pass only the cp.value

Reply:
Actually I think we don't need that method anymore, since we are changing the way of calculating the duals